File: src/java/Area.java
import java.io.*;
class Area
{
	public static void main(String args[])throws IOException
	{
		int side,area;
		DataInputStream dis=new DataInputStream(System.in);
		System.out.println("Enter the side");
		side=Integer.parseInt(dis.readLine());
		area=side*side;
		System.out.println("The area of the square is "+area);

		int length,breadth;
		System.out.println("Enter the length of the rectangle");
		length=Integer.parseInt(dis.readLine());
		System.out.println("Enter the breadth of the rectangle");
		breadth=Integer.parseInt(dis.readLine());
		area=length*breadth;
		System.out.println("The area of the rectangle is "+area);
	}
}

File: docs/c/area.c
/* Area of a square and rectangle */
#include<stdio.h>
int main()
{
	int side,area;
	printf("Enter the side of a square \n");
	scanf("%d",&side);
	area=side*side;
	printf("The area of a square is %d \n",area);

	int length,breadth;
	printf("Enter the length of the rectangle\n");
	scanf("%d",&length);
	printf("Enter the breadth of the rectangle\n");
	scanf("%d",&breadth);
	area=length*breadth;
	printf("The area of a rectangle is %d\n",area);
}